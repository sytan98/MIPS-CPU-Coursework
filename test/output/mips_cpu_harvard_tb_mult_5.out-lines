Output at v0:e38e1c72
