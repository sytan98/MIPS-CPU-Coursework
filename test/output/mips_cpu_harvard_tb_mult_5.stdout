WARNING: ./test/instruction_memory.v:22: $readmemh(./test/cases/mult_5.bytes.txt): Not enough words in the file for the requested range [32:1023].
VCD info: dumpfile mips_cpu_harvard_tb.vcd opened for output.
Register v0:00000000
Register v0:00000000
Register v0:00000000
Register v0:00000000
Register v0:e38e1c72
Register v0:e38e1c72
Output at v0:e38e1c72
TB : finished; running=0
